week-1-practise/level-2/4-currency-conversion-2.js

```diff
@@ -7,8 +7,12 @@
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToUSD(price) {}
-function convertToBRL(price) {}
+function convertToUSD(priceInGBP) {
+  return price * 1.3 * 1.01;
+}
+function convertToBRL(priceInGBP) {
+  return price * 3.78 * 1.01;
+}
 
```

Comment:
Whilst the functions work as intended, it would be worth you looking at the part of the question that asks you to "Find a way to add 1% to all currency conversions (think about the DRY principle)" - have you met that requirement, or does it need a little more thought?